VP9: Added VP9 IVF container detection and picture data parsing support to bitstream reader. (#519)
ROCm/rocDecode: CHANGELOG.md

Full documentation for rocDecode is available at [https://rocm.docs.amd.com/projects/rocDecode/en/latest/](https://rocm.docs.amd.com/projects/rocDecode/en/latest/)

> ## (Unreleased) rocDecode 0.11.0

Reviewer: please also change the rocDecode version [here](https://github.com/ROCm/rocDecode/blob/develop/CMakeLists.txt#L42) and [here](https://github.com/ROCm/rocDecode/blob/develop/api/rocdecode_version.h#L38) too

Author: Done.

### Added

> * The new bitstream reader feature. The bitstream reader contains a few built-in stream file parsers, including elementary stream file parser and IVF container file parser. Currently the reader can parse AVC/HEVC/AV1 elementary stream files, and AV1/VP9 IVF container files. More format support will be added in the future.

Reviewer 2: ```suggestion
* The new bitstream reader feature. The bitstream reader contains built-in stream file parsers, including an elementary stream file parser and an IVF container file parser. It can parse AVC/HEVC/AV1 elementary stream files and AV1/VP9 IVF container files. Additional format support will be added in the future.
```

Author: Suggestion taken. Thanks.
